index.js

```diff
@@ -1 +1,37 @@
+const button = document.querySelector("button");
+const counter = document.querySelector("select");
+const cardsList = document.getElementById("cards-list");
 
+for (let i = 1; i <= 10; i += 1) {
+  const option = document.createElement("option");
+  option.text = i;
+  option.value = i;
+  counter.appendChild(option);
+}
```

Comment:
We would also accept people manually adding the `<option>` elements to the html directly. No need to change this though

Reply:
I can add a comment :)